dapps/marketplace/src/utils/distanceToNow.js

```diff
@@ -1,9 +1,13 @@
-export default function distanceToNow(timestamp, long) {
+import { fbt } from 'fbt-runtime'
+
+export default function distanceToNow(timestamp, { long, showJustNow } = {}) {
   const now = new Date().getTime()
   if (typeof timestamp === 'string') timestamp = Number(timestamp)
   const diff = now / 1000 - timestamp
   if (diff < 60) {
-    return `<1${long ? ' minute' : 'm'}`
+    return showJustNow
+      ? fbt('Just now', 'time.justNow')
+      : `<1${long ? ' minute' : 'm'}`
```

Comment:
Just a very minor nitpick, Shouldn't this be translated?

Reply:
yup good catch 👍